pkg/operator/controller/instance_controller.go

```diff
@@ -134,6 +136,11 @@ func (r *InstanceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 		if err := r.deleteFinalizer(ctx, instance); err != nil {
 			return ctrl.Result{}, err
 		}
+
+		delete(r.InstancesMeta, instance.Spec.Namespace)
+		if err := r.triggerAnotherInstance(ctx, instance.Name, instance.Spec.Namespace); err != nil {
```

Comment:
shouldn't this trigger without `instance.Name` which was just deleted? Isn't it enough to just call with the namespace?
Or maybe the instance with `Name` has not been deleted yet and thus should be ignored in the retrigger?

Reply:
Exactly, is not been deleted yet. (see comment in line 867).